src/Hl7.Fhir.Specification.Tests/Hl7.Fhir.Specification.Tests.csproj

```diff
@@ -63,6 +63,18 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
   </ItemGroup>
+  
```

Comment:
This is not needed, because the ItemGroup about it, will handle this. See 
```
<ItemGroup>
    <Content Include="TestData\**\*.xml;TestData\**\*.json">
      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
    </Content>
  </ItemGroup>
```